assets/webconfig/i18n/de.json

```diff
@@ -466,6 +466,8 @@
     "edt_dev_spec_intervall_title": "Intervall",
     "edt_dev_spec_invert_title": "Invertiere Signal",
     "edt_dev_spec_latchtime_title": "Sperrzeit",
+    "edt_dev_spec_globalBrightnessControlMaxLevel_title": "Maximalstufe Stromstärke",
```

Comment:
I know it is convenient already providing non-EN languages, but we try to do the translations via POEditor.